src/release/main.d

```diff
@@ -893,7 +931,10 @@ Version autodetectVersions ( Version[] tags )
 
     enforce(detected);
 
-    writefln("We are on branch %s", current);
+    if (rel_ver.type == rel_ver.type.Patch)
+        enforce(!options.pre_release,
+            "Prerelease is only for non-patch releases allowed!");
```

Comment:
I would use "Pre-release" as this is how is mentioned in semver (and is easier to read). Also maybe "Pre-releases are only allowed for upcoming minors and majors (not patch releases)".

I wonder if this limitation is really needed though. I think it was for some reason but I don't remember why.